secant/Features/BackupFlow/RecoveryPhraseValidation.swift

```diff
@@ -40,12 +34,8 @@ struct RecoveryPhraseValidationState: Equatable {
     var validationWords: [ValidationWord]
     var route: Route?
     
-    var step: Step {
-        guard !validationWords.isEmpty else { return  .initial }
-
-        guard validationWords.count >= missingIndices.count else { return .incomplete }
-
-        return .complete
+    var isComplete: Bool {
+        !validationWords.isEmpty && validationWords.count >= missingIndices.count
```

Comment:
Should this be >= or just == ?  When would the validation words have a larger number than the missing indeces count?

Reply:
changed